### PR TITLE
IDE-4466 add legacy to new sdk releated wizards

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

**Comment by yanan** [comment]
ft retest

**Comment by yanan** [comment]
ft retest

File: tools/plugins/com.liferay.ide.service.ui/src/com/liferay/ide/service/ui/wizard/NewServiceBuilderWizard.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
 createNewServiceBuilderXmlFile=Create a new service builder xml file in a project.
 newLiferayServiceBuilder=New Liferay Service Builder
-newServiceBuilder=New Service Builder
+newServiceBuilder=New Service Builder (Legacy)
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.service.ui/src/com/liferay/ide/service/ui/wizard/NewServiceBuilderWizard.properties]: Service builder isnt' legacy . It still applies for even new projects.

**Comment by yanan** [comment]
Got it, will resend.

**Pull request closed without merging**